eth-contracts/package.json

```diff
@@ -7,6 +7,7 @@
   "scripts": {
     "ganache": "docker run --name audius_ganache_cli_eth_contracts -d -p 8546:8545 trufflesuite/ganache-cli:v6.9.1 -h 0.0.0.0 -a 50",
     "ganache-test": "docker run --name audius_ganache_cli_eth_contracts_test -d -p 8556:8545 trufflesuite/ganache-cli:v6.9.1 -h 0.0.0.0 -a 50",
+    "ganache-local-network": "docker run --name audius_ganache_cli_eth_contracts -d -p 8546:8545 --network=audius_dev trufflesuite/ganache-cli:v6.9.1 -h 0.0.0.0 --acctKeys eth-contracts-ganache-accounts.json -a 100",
```

Comment:
@SidSethi this will probably break our commit hash but we do actually need the acctKeys / ganche-local-network cmd here. not sure why this command got removed from master in the merge though?

Reply:
a way to preserve the eth-contracts dir hash would be to just issue this cmd manually from service-commands

Reply:
^ im ok doing that. @dmanjunath  / @SidSethi  what's your preference here?

Reply:
doing it through service commands makes sense since this is the command for local setup and has the network name in the cmd

Reply:
makes sense to me to call from service-commands

Reply:
@raymondjacobson yeah pulling it out of both to keep them consistent would be good

Reply:
i want to confirm my assumptions are correct here - should we really be worried about the hash changing for any non-solidity files? there is a downside to doing this, which eliminates the npm command from within the repo

Reply:
i think it's fine to remove it tbh. the audius_dev network is a pretty specific and maybe doesn't even belong in protocol

Reply:
https://github.com/AudiusProject/audius-tooling/pull/33

Reply:
agreed @raymondjacobson. i think best thing here is to not touch eth-contracts at all